server/i18n/en.json

```diff
@@ -4926,6 +4926,22 @@
     "id": "app.custom_group.unique_name",
     "translation": "group name is not unique"
   },
+  {
+    "id": "app.delete_scheduled_post.delete_error",
+    "translation": "Failed to delete scheduled post from database."
+  },
+  {
+    "id": "app.delete_scheduled_post.delete_permission.error",
+    "translation": "You do not have permission to delete this resource."
```

Comment:
Since we are translating, I guess these will end up being user facing strings, so... not sure if `resource` is the right terminology for end user.

I guess "You do not have permission to delete this scheduled post." sounds better in this case.

Reply:
Aren't these just returned by the API? If so I don't see people reading into those errors in the Network tab very often :P 0/5 though

Reply:
@devinbinnie I'll be displaying errors returned from server when user attempts to delete a post.